drools-model/drools-model-compiler/src/main/java/org/drools/modelcompiler/builder/ModelBuilderImpl.java

```diff
@@ -68,6 +84,8 @@ public class ModelBuilderImpl<T extends PackageSources> extends KnowledgeBuilder
 
     private Map<String, CompositePackageDescr> compositePackagesMap;
 
+    private Map<String, List<String>> includedRuleNameMap = new HashMap<>();
```

Comment:
Nitpick: it would be a bit better to replace the `List` with a `Set` for the values of this Map. In this way the `contains` performed [here](https://github.com/kiegroup/drools/pull/59/files#diff-1edc23cad73ee1cb81406436cbdc8e1b656a6e08c793f32c9b0d591131669370R354) would use the String hashing instead of doing a linear search.

Reply:
Thank you. Fixed.